atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/TableFactoryRenderer.java

```diff
@@ -238,6 +246,18 @@ private MethodSpec getTableMethod(String name, TableDefinition tableDefinition)
         return tableGetterMethodBuilder.build();
     }
 
+    private MethodSpec getSimpleTableMethod(String name, TableDefinition tableDefinition) {
```

Comment:
we should probably standardize on `v2` vs `simple` - my preference is for `v2` but don't feel too strongly either way

Reply:
I agree, went with V2 in the end.